### PR TITLE
Minor text/btn changes

### PR DESCRIPTION
**Pull request by benkeen**
This makes a few further additions to buttons / text / styles,
as per Justin's recommendations.

### CODE REVIEW

File: app/addons/fauxton/components.react.jsx [hunk 1]
```diff
@@ -377,6 +378,8 @@ function (app, FauxtonAPI, React, ReactDOM, ZeroClipboard, ReactBootstrap) {
     },
 
     render: function () {
+      var btnClasses = 'btn js-btn-success ' + this.props.buttonClass;
```
Comment (robertkowalski) [inline app/addons/fauxton/components.react.jsx]: `js-btn-success` i think this class is unused

File: app/addons/documents/doc-editor/components.react.jsx [hunk 2]
```diff
@@ -143,12 +143,14 @@ define([
     },
 
     render: function () {
+      var saveButtonLabel = (this.props.isNewDoc) ? 'Create Document' : 'Save Changes';
```
Comment (robertkowalski) [inline app/addons/documents/doc-editor/components.react.jsx]: minor: no parens needed, but feel free to leave it like it is, no hard opinions here from my side

**Comment by robertkowalski** [comment]
good work! +1 after removing the dead class

**Comment by benkeen** [comment]
Merged as bd7205f45c2c2448da7e8e06f743f2d0444177f9

**Pull request closed without merging**